src/Okta.Sdk.IntegrationTests/ApplicationScenarios.cs

```diff
@@ -8,14 +8,6 @@
 using System.Linq;
 using System.Threading.Tasks;
 using FluentAssertions;
-using Org.BouncyCastle.Asn1.X509;
-using Org.BouncyCastle.Crypto;
-using Org.BouncyCastle.Crypto.Generators;
-using Org.BouncyCastle.Crypto.Prng;
-using Org.BouncyCastle.Math;
-using Org.BouncyCastle.Security;
-using Org.BouncyCastle.Utilities;
-using Org.BouncyCastle.X509;
```

Comment:
Since we are not planning to use this library, let's remove the NuGet dependency as well.